assets/cluster-monitoring-configmap.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusOperator:
+      baseImage: quay.io/coreos/prometheus-operator
+      prometheusConfigReloaderBaseImage: quay.io/coreos/prometheus-config-reloader
+      configReloaderBaseImage: quay.io/coreos/configmap-reload
+    prometheusK8s:
+      retention: 24h
+      baseImage: quay.io/prometheus/prometheus
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
```

Comment:
Are we still using infra node in 4.0, I doubt that.

Reply:
Yes. The subscription model has not changed.